Okta/OktaTokenManager.swift

```diff
@@ -21,15 +21,13 @@ open class OktaTokenManager: NSObject, NSCoding {
     open var accessToken: String? {
         // Return the known accessToken if it hasn't expired
         get {
-            guard let tokenResponse = self.authState.lastTokenResponse,
-                  let token = tokenResponse.accessToken,
-                  let tokenExp = tokenResponse.accessTokenExpirationDate,
-                  tokenExp.timeIntervalSince1970 > Date().timeIntervalSince1970 else {
-                    return nil
-            }
-            return token
+            return self.authState.lastTokenResponse?.accessToken
         }
     }
+    
+    open var accessTokenExpirationDate: Date? {
+        return self.authState.lastTokenResponse?.accessTokenExpirationDate
+    }
 
     open var idToken: String? {
```

Comment:
Lets also do the same for our computed `idToken` value. Since we allow passing an expired `idToken` to the `/logout` endpoint, we need this this value instead of returning `nil`.

Reply:
Reworked #87 to get idToken from authState.lastTokenResponse.
Do I still have to make this change?

Reply:
Thanks @ayurok! With your change in #87, this PR is no longer needed.